cmd/snap-confine/snap-confine.apparmor.in

```diff
@@ -275,6 +275,10 @@
     /var/lib/snapd/hostfs/var/lib/lxd r,
 
     # support for the mount namespace sharing
+    capability sys_ptrace,
+    # allow snap-confine to read /proc/1/ns/mnt
+    ptrace peer=unconfined,
```

Comment:
Thank you for the review. I'll tighten the ptrace rule and see if I need sys_ptrace with the fixed kernel.